src/app/socialMedia/social-media-list/social-media-list.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { ApiService } from '../../services/api.service';
+import { SocialMedia } from '../../shared/social-media';
+import { SocialMediaType } from '../../shared/social-media-type';
+
+@Component({
+  selector: 'app-social-media-list',
+  templateUrl: './social-media-list.component.html',
+  styleUrls: ['./social-media-list.component.scss']
+})
+export class SocialMediaListComponent implements OnInit {
+
+  public socialmedia: SocialMedia[];
```

Comment:
Is this accessed outside of the component? Same for the other public field? If so maybe we should use a getter and make that public instead and keep the class fields private.